DateTimes/DateTimes/Program.cs

```diff
@@ -0,0 +1,262 @@
+
+        public static void GetTimeInMiliseconds()
+        {
+            Console.WriteLine("\nExercise 3:");
+
+            Console.WriteLine(Math.Floor((DateTime.Now - DateTime.Now.Date).TotalMilliseconds));
+        }
+
+        // Write a program to get the dates 30 days before and after from the current date, and display it to the console
+
+        public static void GetTheDates30BeforeAndAfterCurrentDate()
+        {
+            Console.WriteLine("\nExercise 4: ");
+
+            Console.WriteLine(DateTime.Now.AddDays(30).ToLongDateString());
+            Console.WriteLine(DateTime.Now.AddDays(-30).ToLongDateString());
+        }
+
+        // Write a program to calculate a number of days between two dates.
+
+        public static void CalculateNumberOfDaysBetweenTwoDates()
+        {
+            Console.WriteLine("\nExercise 5: ");
+
+            DateTime firstDate = new DateTime(1988, 4, 10);
+            DateTime secondDate = new DateTime(2000, 1, 1);
+            TimeSpan numberOfDaysBetweenDates = secondDate - firstDate;
+            Console.WriteLine(numberOfDaysBetweenDates.TotalDays);
+        }
+
+        // Write a program that calculates the date six months from the current date.
+
+        public static void GetDateSixMonthFromCurrent()
+        {
+            Console.WriteLine("\nExercise 6: ");
+
+            Console.WriteLine(DateTime.Now.AddMonths(6).ToLongDateString());
+        }
+
+        // Write a program to add 5 seconds to the current time and print it to the console.
+
+        public static void AddFiveSecondsToCurrentTime()
+        {
+            Console.WriteLine("\nExercise 7: ");
+
+            Console.WriteLine(DateTime.Now.AddSeconds(5));
+        }
+
+        // Write a program to print yesterday, today, tomorrow.
+
+        public static void PrintYesterdayTodayTomorrow()
+        {
+            Console.WriteLine("\nExercise 8: ");
+
+            Console.WriteLine("Today : {0}", DateTime.Now.ToShortDateString());
+            Console.WriteLine("Yesterday : {0}", DateTime.Now.AddDays(-1).ToShortDateString());
+            Console.WriteLine("Tomorrow : {0}", DateTime.Now.AddDays(1).ToShortDateString());
+        }
+
+        // Write a program to add N year(s) to the current date and display the new date.
+
+        public static void AddYearsToCurrentDate()
+        {
+            Console.WriteLine("\nExercise 9: ");
+
+            Console.WriteLine("Please enter a number, that represents the number of years that you want to be added to the current date.");
+            int yearsToAdd = int.Parse(Console.ReadLine());
+
+            Console.WriteLine(DateTime.Now.AddYears(yearsToAdd).ToShortDateString());
+        }
+
+        // Write program to print next 5 days starting from today.
+
+        public static void GetNextFiveDaysFromToday()
+        {
+            Console.WriteLine("\nExercise 10: ");
+
+            Console.WriteLine(DateTime.Now.DayOfWeek);
```

Comment:
aici mai puteai face cu un for, si adaugai i-ul. Nu e gresit nici cum ai facut tu, era doar mai putin duplication